src/All/All.csproj

```diff
@@ -17,8 +17,6 @@
 
     <PackageReference Include="GraphQL.MemoryCache" Version="$(GraphQLVersion)" />
     <PackageReference Include="GraphQL.DataLoader" Version="$(GraphQLVersion)" />
-    <PackageReference Include="GraphQL.MicrosoftDI" Version="$(GraphQLVersion)" />
```

Comment:
I am curious here. Create simple console net5/net6 app, then reference this metapackage. What version (I mean what TFM) of GraphQL.MemoryCache/GraphQL dlls will you see in output folder? netstandard2.0 of net5/net6 ?

Reply:
I created a console app only referencing the All meta-package.  I then verified that the DLLs are the appropriate version with a binary file comparison.  Specifically, GraphQL.SystemTextJson (pulled from nuget) is the net6 version (of version 6.0.0-preview-602) and Transports.AspNetCore (pulled from the local build) is the net5 version.  I did not check other files.

Reply:
OK. So TFM resolution works "globally" and is not constrained by consuming package.